app/assets/stylesheets/application.scss

```diff
@@ -199,13 +199,20 @@ a:hover {
 }
 
 .actions input {
+  font-family: $nexa-font;
```

Comment:
Properties should be ordered background, border, color, font-family, font-size, padding-bottom